packages/forms-web-app/src/locales/en/common.json

```diff
@@ -75,6 +76,14 @@
 		"aboutProject": {
 			"empty": "Enter what you want to tell us about this proposed project",
 			"length": "What you want to tell us must be {{-maxCharacters}} characters or less"
+		},
+		"organisationName": {
+			"empty": "Enter your organisation or charity name",
+			"length": "Name of your organisation or charity name must be 255 characters or less"
+		},
+		"jobTitleOrRole": {
+			"empty": "Enter your job title or volunteer role",
+			"length": "Your job title or volunteer role must be 64 characters or less"
 		}
```

Comment:
Would these translations be better placed in a common register folder scoped to the journey?

Reply:
good idea - I'll move them